core/src/test/scala/org/apache/spark/sql/delta/DeltaConfigSuite.scala

```diff
@@ -158,4 +158,43 @@ class DeltaConfigSuite extends SparkFunSuite
       assert(e.getMessage == msg)
     }
   }
+
+  test("Allow setting Serializable Isolation Level") {
```

Comment:
nit: could we just update this test name to reflect the current test? also generally we don't capitalize test names (it seems this specific suite isn't consistent with that)

Reply:
broke it up so tests could have better names